components/modules/MentorCard/variants/Desktop.tsx

```diff
@@ -1,169 +1,11 @@
 import "twin.macro";
```

Comment:
The card doesn't have much responsiveness past a certain threshold, is that going to be a future story?